config/boards/shields/artboard/README.md

```diff
@@ -0,0 +1,3 @@
+# Artboard
+
+This directory contains the ARTSEY implementation for the `Artboard` hardware being developed by depadiernos.
```

Comment:
Can you add a link to the repo where the artboard lives in case people want to get hands on the pcb / design for fab?